### ISSUE TITLE
[Basic (?) Feature Request] Allow Client to work with multiple certificates and related keys, chains etc

### ISSUE THREAD

**Issue by dpurnam** [gregtwallace/certwarden#59]
Self explanatory title/Feature Request. The client should be able to pull **multiple** certificates from Cert Warden server. No value add of running multiple containers for clients for each unique certificate.

Thank you for the project, though.

**Comment by kroimon** [comment]
I know it's not the official client, but I found this project that might be helpful to you: [certwarden-deploy](https://certwarden-deploy.adora.codes/)

**Comment by Akruidenberg** [comment]
Any progress? It would be really nice to use multiple certs. I think it can be done with envs?

**Comment by gregtwallace** [comment]
Yes, I started working on this and am nearing a build for testing.

**Comment by gregtwallace** [comment]
v0.5.0 of the client has this added. I did not push the new build to github or dockerhub yet. Please build and test.

**Comment by gregtwallace** [comment]
Great!
